charts/kube-prometheus-stack/values.yaml

```
@@ -2286,6 +2286,14 @@ prometheus:
     ## WebTLSConfig defines the TLS parameters for HTTPS
     ## ref: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#webtlsconfig
     web: {}
+    
+    ## Exemplars related settings that are runtime reloadable.
+    ## It requires to enable the exemplar storage feature to be effective.
+    exemplars:
+      # Maximum number of exemplars stored in memory for all series.
+      # If not set, Prometheus uses its default value.
+      # A value of zero or less than zero disables the storage.
+      maxSize: 100000
```

why not using the default?

```suggestion
      maxSize: ""
```